controllers/pullRequestsController.js

```diff
@@ -43,6 +43,52 @@ const getPRdetails = async (req, res) => {
   }
 }
 
+const getStalePRs = async (req, res) => {
+  try {
+    const { data } = await githubService.fetchOpenPRs()
+
+    if (data.total_count) {
+      const allPRs = []
+      data.items.forEach(({
+        title,
+        html_url: url,
+        state,
+        created_at: createdAt,
+        updated_at: updatedAt,
+        draft,
+        labels,
+        user,
+        assignees
+      }) => {
+        const allAssignees = assignees.map(object => object.login)
+        const allLabels = labels.map(object => object.name)
+        allPRs.push({
+          title,
+          state,
```

Comment:
Do we really need to return the state when we already know it's stale (open) ?

Reply:
Good point. I deliberately kept it there to control the text we will show on the Frontend against that PR row. In future, instead of "Open", if we want to call it "Unattended", we simply need to make backend changes, and when the user sees it will not be affected by the assets caching on browser or CDN